js/wp-seo-admin.js

```diff
@@ -244,3 +244,73 @@ function wpseo_add_fb_admin() {
 		}
 	);
 }
+
+function wpseo_recalculate_scores(current_page) {
```

Comment:
We should start limiting the amount of global scope we are using, start putting all the functions we create in a `WPSEO` javascript object so we don't pollute the global scope.